### PR TITLE
Add new docs for UX changes in VS Code

### PR DESCRIPTION
**Pull request by MaedahBatool**
With the new VS Code release this PR adds all new docs.

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@kalanchan can you please review the docs.

File: docs/cody/clients/install-vscode.mdx [hunk 1]
```diff
@@ -232,10 +231,10 @@ Cody's Enhanced Context enables Cody to leverage search and embeddings-based con
 <Callout type="note">Enhanced context selector offers single-repo support for Cody Free and Pro while multi-repo support for Cody Enterprise.</Callout>
```
Comment (kalanchan) [inline docs/cody/clients/install-vscode.mdx:231]:
```suggestion
<Callout type="note">Cody Free and Pro offers single-repo context, and Cody Enterprise supports multi-repo context.</Callout>
```

File: docs/cody/clients/install-vscode.mdx [hunk 2]
```diff
@@ -207,17 +184,39 @@ When you `@-mention` files to add to Cody’s context window, the file lookup ta
...
+- Current file only: Re-run the prompt again using just the current file as context
+- Add context: Provides @-mention context options to improve the response. Could be files, symbols, remote repositories, or even web URLs
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto', aspectRatio: '1920 / 1080' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/rerun-context-0624.mp4" />
+</video>
 
 ## Context fetching mechanism
 
 VS Code users on the Free or Pro plan can leverage [local context](/cody/core-concepts/context#context-selection) (local keyword search) as the primary context source for Cody chat with access to local embeddings. Remote embeddings will no longer be produced or used as a context source.
```
Comment (kalanchan) [inline docs/cody/clients/install-vscode.mdx:217]: hmm, we should double check this after the release. I'm not sure what happened to embeddings now lol

**Review by kalanchan — APPROVED** [review]
looks good! i left you one comment

**Pull request merged**